plugins/expired_gpg_keys.py

```diff
@@ -0,0 +1,105 @@
+import dnf
+import rpm
+import subprocess
+
+from datetime import datetime
+from dnfpluginscore import _, logger
+
+
+class ExpiredGPGKeys(dnf.Plugin):
+    """
+    Find expired GPG keys and suggest their removal.
```

Comment:
Please replace "GPG" everywhere with "PGP". Now when librpm does not use GnuPG anymore, it's really important to stop spreading the confusion.